datacontract/imports/avro_importer.py

```diff
@@ -91,6 +91,19 @@ def handle_config_avro_custom_properties(field: avro.schema.Field, imported_fiel
             imported_field.config = {}
         imported_field.config["avroDefault"] = field.default
 
+# Mapping for Avro logical types to data contract types
+LOGICAL_TYPE_MAPPING = {
+    "decimal": "decimal",
+    "date": "date",
+    "time-millis": "time",
+    "time-micros": "time",
+    "timestamp-millis": "timestamp_tz",
+    "timestamp-micros": "timestamp_tz",
+    "local-timestamp-micros": "timestamp_tz",
```

Comment:
local-timestamps don't have a timezone, so it would be mapped to `timestamp_ntz`